### DISCUSSION TITLE
Get word count of actually changed words (and not whole source string count)

### DISCUSSION THREAD

**Discussion by 27leaves** [WeblateOrg/weblate#10550] [Q&A] [answered]
Hello!

We're using weblate 4.17 and I have a question regarding word counts: We usually invoice our translators per word count. In our latest developments we changed a lot of texts, but often we only replaced certain terms for others (for example shop -> place because we need it to be a more general term). 

When counting the words I figured that weblate counted all the words within one translation and not only the changed one. Now I counted all the words manually 😅 because otherwise it would be very costly for us.

For example at the component "coupons" the "Unfinished words" display 572 words, but manually I counted 76 changed words.
<img width="1151" alt="image" src="https://github.com/WeblateOrg/weblate/assets/999419/1c8e8c3f-aeef-406a-80a9-f90e0e7faee9">

Here it counts 16 instead of 1:
<img width="843" alt="image" src="https://github.com/WeblateOrg/weblate/assets/999419/656a91ad-47ff-4235-8fe2-0dde5fee1ba2">

Is this a feature that already exists? Thank you for every help

**Comment by nijel** [comment, marked as answer]
For changed strings, there is a count of edits (measured in Damerau–Levenshtein distance) in [contributor stats](https://docs.weblate.org/en/latest/devel/reporting.html#contributor-stats). There is no such word-based measure yet (because word counting is tricky for some languages, see #10278).

**Reply by 27leaves** [reply]
Thank you @nijel!
